src/qt/rpcconsole.cpp

```diff
@@ -906,6 +906,16 @@ void RPCConsole::on_lineEdit_returnPressed()
         return;
     }
 
+    if (cmd == QLatin1String("stop")) {
```

Comment:
b87a4cd6bb58a6458dea7527a19433ead0ea848d

Could add a comment explaining this case.

Reply:
[Updated](https://github.com/bitcoin-core/gui/pull/123#issuecomment-831538996).